sites/dwhdelft.nl/components/JoinOptions.vue

```diff
@@ -2,11 +2,14 @@
 en:
   title: Ready to check out <strong>DWH</strong>?
   description: |-
-    You're always welcome on a Saturday night during a bar night. Do you think this is too exciting or
-    frightening? Then you can make use of our bar buddy system!
-  outsite_hint: |-
-    Are you 28 years or younger? Take a look at
-    <a href="https://outsite.nl/" class="font-bold">Outsite</a>, the youth division of our association.
+    The best way to get to know DWH is at one of our activities. Join us at one, or sign up for a
+    barbuddy so there'll be someone you know. Are you 28 or younger? Do take a look at  
+    <a href="https://outsite.nl/" class="font-bold">Outsite (28-)</a> or 
+    <a href="https://jongenoutdelft.nl/" class="font-bold">Jong&Out Delft (12-18)</a>.
```

Comment:
```suggestion
    <a href="/jongenout" class="font-bold">Jong&Out Delft (12-18)</a>.
```
```suggestion
    <a href="/jongenout" class="font-bold">Jong&Out Delft (12-18)</a>.
```
dan is de navigatie mooier :)